server/discussion/__tests__/api.test.ts

```diff
@@ -17,7 +17,7 @@ const couldApplyManagedLabel = {
 };
 
 const publicLabel = {
-	title: 'Authors can apply this label to their own discussions',
+	title: 'Contributors can apply this label to their own discussions',
```

Comment:
Should match the language you use above.

Reply:
I changed the sentence to the same as the public labels above in order to avoid the ouroboros of discussants applying labels to their discussions. (Sans period to be consistent with adjacent strings.